kombu/transport/redis_cluster.py

```diff
@@ -299,7 +317,12 @@ def close(cls, connection):
             connection.close()
```

Comment:
i don't think closing producer connection is proper action in here

Reply:
Anyway are there any cases where we need to close producer connection when we share with this connection with `SodaCache`?

Reply:
Yes, but there aren't simple way to find if it's shared rediscluster on kombu, so i modified soda's rediscluster to not close on close() call without any argument.